skeleton/templates/cdk/src/api-stack.ts.mustache

```diff
@@ -67,6 +68,8 @@ export class ApiStack extends Stack {
                 enablePerformanceInsights: true,
             }),
             defaultDatabaseName: "api",
+            storageEncrypted: true,
```

Comment:
Isn't Aurora encrypted in rest by default?

Reply:
it is not.